example-site/src/pages/RequestAccess.tsx

```diff
@@ -11,10 +11,18 @@ const requestAccess = async (
   await pod.grantAccess(forWebId, [fileUrl]);
 };
 
+const getAccessGrants = async (
+  pod: UmaPod,
+  forWebId: string
```

Comment:
beetje rare parameter naam mss :p
```suggestion
  webId: string
```

Reply:
ah da's die example site, nie zo belangrijk